src/rust/src/zip339_ffi.rs

```diff
@@ -46,15 +47,15 @@ pub extern "C" fn zip339_entropy_to_phrase(
 ) -> *const c_char {
     assert!(!entropy.is_null());
 
-    if let Ok(language) = language.try_into() {
+    || -> Option<_> {
```

Comment:
Does this need to be a closure, or could it just be a scope? `{...}.unwrap_or(ptr::null())`